### ISSUE TITLE
AutoCluster - Grafana dashboard isn't displaying Status Codes

### ISSUE THREAD

**Issue by daveharris1** [VIP-Logic/NBomber#697]
The following issues has been raised after talking with Anton Moldovan.

NBomber NuGet package version: 5.6.0
NBomber.Sinks.InfluxDB NuGet package version: 5.0.2

When running distributed tests the Grafana dashboard 'Scenario Status Code' panel doesn't display any results.

The Grafana dashboard was created using the **nbomber_board.json** and **nbomber_sessions_board.json** found here:

https://github.com/PragmaticFlow/NBomber.Grafana

![image](https://github.com/PragmaticFlow/NBomber/assets/12339745/de10cd04-896b-4d3b-a977-04e769cadf0d)

Sudo-code example of the code:

```
public async Task ApiResponseTimes(TestData testData)
        {
            var scenario = Scenario.Create("ApiResponseTimes", async context =>
            {
		//test logic
		var apiResponse = await AnApiCall(testData.Url).ConfigureAwait(false);		
		
		//Expect the Status code to be registered and displayed
                return Response.Ok(statusCode: apiResponse.StatusCode.ToString());
            })
            .WithRestartIterationOnFail(false)
            .WithoutWarmUp()
            .WithLoadSimulations(Simulation.KeepConstant(copies: testData.Copies, this.Duration));
            var nbomberRunner = NBomberRunner.RegisterScenarios(scenario)
                .WithLicense(await this.KeyVaultClient.GetSecretStringAsync(this.TestId, "NBomberKey").ConfigureAwait(false))
                .LoadConfig(await this.GenerateConfig().ConfigureAwait(false))
                .LoadInfraConfig(await this.GenerateInfraConfig().ConfigureAwait(false))
                .WithReportingSinks(this.influxDbSink)
                .Run();
```

**Comment by AntyaDev** [comment]
Hi @daveharris1 

The StatusCode and Scenario Latency indicators will be displayed when the test status = complete. For "bombing" phase it's not display data. 

![image](https://github.com/PragmaticFlow/NBomber/assets/1080518/b3861a93-852a-48cf-b5b4-faec49e394ba)

**Comment by AntyaDev** [comment]
You can even find this information on the panel. There is a small button with a hint.

![image](https://github.com/PragmaticFlow/NBomber/assets/1080518/1c7bc789-29b6-4045-b6b6-880fabf041b5)

**Comment by daveharris1** [comment]
> You can even find this information on the panel. There is a small button with a hint.
> 
> ![image](https://private-user-images.githubusercontent.com/1080518/335710854-1c7bc789-29b6-4045-b6b6-880fabf041b5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODYyMjksIm5iZiI6MTcxNzQ4NTkyOSwicGF0aCI6Ii8xMDgwNTE4LzMzNTcxMDg1NC0xYzdiYzc4OS0yOWI2LTQwNDUtYjZiNi04ODBmYWJmMDQxYjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMDcyNTI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWE2OTg4ZTA0ZDk0ODg3ZDM2YzMwYzJkYTljMDhjZWU5N2RjZWEyMDlhOGUwZmZhOWZjMDQ0Njk1ZjAyOGQyMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JA7eKj7Y8RqNJ3TOEi-8y73MMXJ3-Hp__mR0uG5IIyo)

Thanks Anton. In our use case we have a number of StopWatches being used that determine when a distributed AutoCluster run should be stopped early via:

`void StopCurrentTest(string reason);`
(https://nbomber.com/docs/using-nbomber/basic-api/scenario/#scenario-context)

This means that the run ends in the following state of 'None':

![image](https://github.com/PragmaticFlow/NBomber/assets/12339745/7e37507d-22f6-46ce-a0b3-7e1e8d5f0021)

Can we not have these HTTP statuses made available no matter what the state ends in?

**Comment by AntyaDev** [comment]
I think it's possible to overcome

**Comment by AntyaDev** [comment]
Fixed in NBomber [5.7.0-beta.7](https://www.nuget.org/packages/NBomber/5.7.0-beta.7)
